news/2024/2024-05-21-the-perennial-returns.md

```diff
@@ -0,0 +1,64 @@
+
+## Original music and original maps
+
+The Perennial also has a history, albeit a short one, of bringing out the best in our mappers and staff. Many of the custom maps from both 2021 and 2022 are either ranked or loved, despite not being original songs, and many of those with leaderboards are now player favorites. Take [Nathan](https://osu.ppy.sh/users/4785223)'s map of [Angel Salad](https://osu.ppy.sh/beatmapsets/1471137#osu/3020221) by [katagiri](https://osu.ppy.sh/beatmaps/artists/176) as an example.
+
+We release multiple custom maps each year, but in 2022 we were the first tournament to release an original song by a live band. This year, we're upping our past figures with at least one original song every round of the tournament, along with an original background for every original song. You can check out our past original song, The Throne by UNDEAD CORPORATION, in the embed below, and check out past rounds and highlights from 2021 and 2022 on our [YouTube channel](https://youtube.com/c/ThePerennialYT).
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/PCVtU9kWo1w" frameborder="0" allowfullscreen></iframe>
+</div>
+
+## What's new?
+
+While past iterations have run off a standard Round of 32 double elimination bracket with no restrictions on team formation, this year is slightly different. Teams will be limited to players all within one of three regions: the Americas; Europe, the Middle East, and Africa (EMEA); and Asia, Australia, and Oceania (AAO). Further details on this rule and exact definitions for each region can be found on our website.
+
+Additionally, the tournament format will see some slight changes, with qualifiers determining seeding before taking the 9th to 32nd seeds into a single round-robin group stage. The winners of each group will be drafted to face off against the 1st to 8th seeds awaiting them in the Round of 16 double elimination bracket.
+
+You can head over to the **[website](https://www.theperennial.net/)** to find out more details about the overall tournament and mappool structure as well as other relevant rules.
+
+## Tournament schedule
+
+| Event | Dates |
+| --: | :-- |
+| Registration phase | May 20 - June 9 |
+| Qualifier showcase | June 9, 20:00 UTC |
+| Qualifier stage | June 15 - 16 |
+| Group Stage | June 22 - 23 |
+| RO16 & QF | July 29 - 30 |
+| Semi Finals | July 6 - 7 |
+| Finals | July 13 - 14 |
+| Grand Finals | July 20 - 21 |
+
+## Prizes
+
+| Placement | Badge | Trophy | Cash |
+| --: | :--: | :--: | :-- |
+| First | &check; | &check; | 50% |
+| Second | &check; | &check; | 33% |
+| Third | &check; | &check; | 17% |
```

Comment:
while this "works" for news posts specifically, it wouldn't work for wiki articles where we simply put in the characters directly anyway

(also ci complains)

```suggestion
| Placement | Badge | Trophy | Cash |
| --: | :-: | :-: | :-- |
| First | ✓ | ✓ | 50% |
| Second | ✓ | ✓ | 33% |
| Third | ✓ | ✓ | 17% |
```